client/src/pages/Home.js

```diff
@@ -21,18 +21,19 @@ const StyledIntro = styled.div`
   }
 
   @media screen and (min-width: ${mq.tablet.wide.minWidth}) {
```

Comment:
I'm not sure which media query it is but the page looks kinda off between 450px and 1024px. I think the desktop version shows until 450px? Right now it stops showing <1024px


<img width="1068" alt="Screen Shot 2020-05-10 at 11 52 37 PM" src="https://user-images.githubusercontent.com/42228386/81533073-66b82b80-931a-11ea-97d1-c33168e36c0c.png">

Reply:
I haven't been able to figure out what's going on here yet...

Reply:
Yea, this is also happening in #322. I think it has to do with a media query controlling the blue box. It looks like the media query is hiding the box at 1024px (too early) instead of 450px

Reply:
Someone just opened an issue for that. I'll leave fixing this problem to the other ticket.